### DISCUSSION TITLE
Nextcloud 24.0.1 has been out, Are we waiting for 24.0.2?

### DISCUSSION THREAD

**Discussion by johnnyq** [nextcloud/all-in-one#774] [Questions] [answered]
Will the next Nextcloud AIO be released with 24.0.1? or are we waiting for 24.0.2 to come out?

**Comment by szaimen** [comment, marked as answer]
Hello, see https://github.com/nextcloud/all-in-one/issues/669

**Comment by johnnyq** [comment]
I see there are still some high priority outstanding issues Thanks @szaimen

**Comment by szaimen** [comment]
This is now released with 1.7.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
